### PR TITLE
Use navigator.clipboard API when available

### PR DESCRIPTION
**Pull request by architjain**

### CODE REVIEW

File: src/js/components/CopyToClipboard.js [hunk 1]
```diff
@@ -25,21 +25,32 @@ export default class extends React.PureComponent {
...
+        } else {
+            const container = document.createElement('textarea');
+            container.innerHTML = textToCopy;
+            document.body.appendChild(container);
+            container.select();
+            document.execCommand('copy');
+            document.body.removeChild(container);
+            
+        }
+
```
Comment (Kikobeats) [inline src/js/components/CopyToClipboard.js:40]:
```suggestion
        return await navigator.clipboard.writeText(textToCopy);
```

I think it's ok to assume `navigator.clipboard.writeText` is there

compat is +90% https://caniuse.com/mdn-api_clipboard_writetext

Comment (mbtools) [reply src/js/components/CopyToClipboard.js:40]: agree, but then you can just replace `await copyText` with `await navigator.clipboard.writeText(textToCopy);`

**Comment by SimonSchick** [comment]
I assume this would fix https://github.com/mac-s-g/react-json-view/issues/131 ?

**Comment by mbtools** [comment]
I believe so, @SimonSchick

**Comment by Kikobeats** [comment]
Happy to merge this if someone update the PR with the suggested code 🙏

**Comment by mbtools** [comment]
I would but am not authorized to change the PR. @architjain

**Comment by Kikobeats** [comment]
@mbtools just a new PR 🙂

**Comment by Kikobeats** [comment]
superseed by #39

**Pull request closed without merging**